src/auto_archiver/modules/gsheet_feeder_db/gsheet_feeder_db.py

```diff
@@ -101,6 +110,7 @@ def started(self, item: Metadata) -> None:
         logger.info(f"STARTED {item}")
         gw, row = self._retrieve_gsheet(item)
         gw.set_cell(row, "status", "Archive in progress")
+        logger.info(f" row: {row} on {gw.wks.spreadsheet.title} : {gw.wks.title}")
```

Comment:
ignored

Reply:
This is quite useful for me as it enforces a folder name is given, thus the S3/Gdrive storage remains organised in a folder structure. Thought it may be useful to have in the codebase for others.

Reply:
the ignore is just about the log